diff --git a/node_modules/react-native-pdf/android/build.gradle b/node_modules/react-native-pdf/android/build.gradle
index dcbc755..1d50159 100644
--- a/node_modules/react-native-pdf/android/build.gradle
+++ b/node_modules/react-native-pdf/android/build.gradle
@@ -2,6 +2,7 @@ description = 'react-native-pdf'
 
 buildscript {
     repositories {
+        mavenCentral()
         jcenter()
         google()
     }
@@ -12,6 +13,7 @@ buildscript {
 }
 
 repositories {
+    mavenCentral()
     jcenter()
     maven {
         url 'https://jitpack.io'
